### ISSUE TITLE
Login window too small, cant resize

### ISSUE THREAD

**Issue by jellopudding** [FAForever/downlords-faf-client#2993]
**Describe the bug**
When launching a fresh update the client will request login, however this window is too small to interact with and is not resizable in order to become intractable. Also, oddly, it is impossible to close the window without using task manager because 1. the window has no Default UI elements (title bar, minimize, maximize, close) and 2. the two methods of closing using the task bar do not function.

**To Reproduce**
1. Update the client to newest version (as of 6/16/23)
2. Run the launcher after update

**Log or error message**
This is my log file:
[client.log](https://github.com/FAForever/downlords-faf-client/files/11772581/client.log)

**Expected behavior**
Login window is at least large enough to interact with or resizable

**Screenshots**
![The entire login window](https://github.com/FAForever/downlords-faf-client/assets/5677453/545414e1-8dd0-43c9-bf3b-09fe7ef7619a)

**OS**
Windows 10 Pro, 10.0.19045 Build 19045

**Comment by Sheikah45** [comment]
Likely something messed with the starting window size in your client.prefs file. You can try renaming or deleting the prefs file

**Comment by jellopudding** [comment]
> Likely something messed with the starting window size in your client.prefs file. 

Indeed, something was, `"width"` was set to 160 and `"height"` was set to 39 under `"mainWindow"`. I'm not sure how that happened.

**Comment by Nerdboy2** [comment]
I got the exact same issue after the latest update. 
Thanks for leaving this up.
